<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vehiculewin</class>
 <widget class="QDialog" name="Vehiculewin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">vehiculewin {
background: blue;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1201</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: 3px solid gray;
border-radius: 40px;
background: rgb(255, 255, 127);
}
</string>
   </property>
   <property name="title">
    <string>Les vehicules</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid gray;
border-radius: 40pxt;
background: white;
}
QDateEdit {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid gray;
border-radius: 40pxt;
background: white;
}
QComboBox {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid gray;
border-radius: 40pxt;
background: white;
}
QPushButton
{
 color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}
QTabWidget{

adding: 5px;
border-radius: 5px;
background-color:#d7caff;
font-weight: bold;

}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter </string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Numero de chassis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numerochassis">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Marque de vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="marque">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Date de circulation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="datecirculation">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type de vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typev">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type carburant&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typecarburant">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>310</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Id employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idemp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Matricule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Matricule">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>421</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{

adding: 5px;
border-radius: 5px;
background-color:#d7caff;
font-weight: bold;
}

</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>label_6</zorder>
     <zorder>numerochassis</zorder>
     <zorder>label_2</zorder>
     <zorder>marque</zorder>
     <zorder>label_5</zorder>
     <zorder>datecirculation</zorder>
     <zorder>label_3</zorder>
     <zorder>typev</zorder>
     <zorder>label_7</zorder>
     <zorder>typecarburant</zorder>
     <zorder>Ajouter</zorder>
     <zorder>label_8</zorder>
     <zorder>idemp</zorder>
     <zorder>label</zorder>
     <zorder>Matricule</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Matricule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Matricule_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Numero de chassis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numerochassis_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Marque de vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="marque_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Date de circulation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="datecirculation_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type de vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typev_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type carburant&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typecarburant_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Id employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idemp_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{

adding: 5px;
border-radius: 5px;
background-color:#d7caff;
font-weight: bold;
}

</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <zorder>groupBox_3</zorder>
     <zorder>label_11</zorder>
     <zorder>Matricule_2</zorder>
     <zorder>label_12</zorder>
     <zorder>numerochassis_2</zorder>
     <zorder>label_13</zorder>
     <zorder>marque_2</zorder>
     <zorder>label_14</zorder>
     <zorder>datecirculation_2</zorder>
     <zorder>label_15</zorder>
     <zorder>typev_2</zorder>
     <zorder>label_16</zorder>
     <zorder>typecarburant_2</zorder>
     <zorder>label_17</zorder>
     <zorder>idemp_2</zorder>
     <zorder>Modifier</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Donner la matricule de vehicule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supprimer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>411</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{

adding: 5px;
border-radius: 5px;
background-color:#d7caff;
font-weight: bold;
}

</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>label_9</zorder>
     <zorder>supprimer</zorder>
     <zorder>supprimer_2</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Afficher les vehicules&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tabvehicule">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>721</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 3px solid gray;
border-radius: 40pxt;
background: white;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
